src/polycubed/src/port.cpp

```diff
@@ -141,8 +137,18 @@ void Port::set_peer(const std::string &peer) {
   }
   ServiceController::set_port_peer(*this, peer);
 
-  std::lock_guard<std::mutex> guard(port_mutex_);
-  peer_ = peer;
+  {
+    std::lock_guard<std::mutex> guard(port_mutex_);
+    peer_ = peer;
+  }
+
+  // Check if peer is a ExtIface
+  ExtIface* extiface = dynamic_cast<ExtIface*>(get_peer_iface());
+  if (extiface) {
+    // Subscribe to the list of events
+    for (auto it : subscription_list)
+      extiface->subscribe_parameter(name_, it.first, it.second);
```

Comment:
`name_` is being used as caller_id here, this is wrong because this is possible to have a transparent cube and a port with the same name. Maybe this is a good place to use the uuid of those elements, so we don't have to bother about possible collisions.